test/e2e/azure_lb.go

```diff
@@ -113,7 +114,7 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 
 	// TODO: fix and enable this. Internal LBs + IPv6 is currently in preview.
 	// https://docs.microsoft.com/en-us/azure/virtual-network/ipv6-dual-stack-standard-internal-load-balancer-powershell
-	if !input.IPv6 {
+	if !input.IPv6 && !input.IsVMSS {
```

Comment:
this could lead to a bug if we reuse `IsVMSS` for something else in the file since it's not actually being set to true for all VMSS specs. Consider either 1) renaming this to `isWindowsVMSS`, or 2) setting `isVMSS` on all VMSS specs like before and changing this condition to `if !input.IPv6 && !(input.IsVMSS && input.IsWindows) {`

Reply:
will use isWindowsVMSS since it supposed to be temporary